builds/mariner/SPECS/rust/rust.spec

```diff
@@ -0,0 +1,60 @@
+Summary:        Rust Programming Language
+Name:           rust
+Version:        1.45.2
```

Comment:
Same here. The version of Rust that is built must be the exact version listed in `edgelet/rust-toolchain`.